src/lib/opts.rs

```diff
@@ -346,12 +360,12 @@ impl Opts {
                     Some(Ok(record)) => {
                         if first_head.is_empty() {
                             first_head = record.head().to_vec();
-                            end_index = first_head.find_byte(b' ').unwrap_or(first_head.len());
+                            end_index = first_head.find_byte(b':').unwrap_or(first_head.len());
```

Comment:
I think this was a bug, since the lane is in the read name prior to the first space.